Makefile

```diff
@@ -363,6 +363,7 @@ test/k3s-%: dist/img.tar bin/k3s dist-%
 	sleep 5s
 	sudo bin/k3s kubectl wait deployment wasi-demo --for condition=Available=True --timeout=5s
 	sudo bin/k3s kubectl get pods -o wide
+	sleep 300s
```

Comment:
I think we have a 5m timeout in CI, which this would exceed. Does it need to be that long?

Reply:
Something I was testing locally got mixed in 🙏